packages/app/src/cli/services/dev/extension/bundler.ts

```diff
@@ -183,12 +188,15 @@ export async function setupConfigWatcher({
   stderr,
   signal,
   specifications,
+  unifiedDeployment,
 }: SetupConfigWatcherOptions) {
   const {default: chokidar} = await import('chokidar')
 
   const configWatcher = chokidar.watch(extension.configurationPath).on('change', (_event, _path) => {
     outputInfo(`Config file at path ${extension.configurationPath} changed`, stdout)
-    updateExtensionConfig({extension, token, apiKey, registrationId, stderr, specifications}).catch((_: unknown) => {})
+    updateExtensionConfig({extension, token, apiKey, registrationId, stderr, specifications, unifiedDeployment}).catch(
+      (_: unknown) => {},
+    )
   })
```

Comment:
Did you test that the configWatcher also work for functions?
Maybe the functions watcher should take care of watching the `.toml` file as well, to take advantage of the buildController 🤔

Reply:
Actually no, I did not test this. Tue that we could end up with dueling deploys and the config update here will push a new wasm blob. Changing the toml doesn't usually require a build, but maybe we could special case that in the function watcher?